eZ/Bundle/EzPublishRestBundle/Resources/config/input_parsers.yml

```diff
@@ -67,6 +67,16 @@ parameters:
 
     ezpublish_rest.input.parser.internal.sortclause.data_key_value_object.class: eZ\Publish\Core\REST\Server\Input\Parser\SortClause\DataKeyValueObjectClass
 
+    ezpublish_rest.input.parser.internal.facetbuilder.content_type.class: eZ\Publish\Core\REST\Server\Input\Parser\FacetBuilder\ContentTypeParser
```

Comment:
Open question:
In Symfony `3.0` they removed `.class` parameters, placing class names directly in the service definition.
Maybe we should do the same for these parameters that are not reusable in other places?
WDYT @andrerom @Nattfarinn @bdunogier ?